client/profile-wizard/index.js

```diff
@@ -85,20 +83,6 @@ class ProfileWizard extends Component {
 		recordEvent( 'storeprofiler_step_view', {
 			step: this.getCurrentStep().key,
 		} );
-
-		// Track plugins if already installed.
-		if (
-			activePlugins.includes( 'woocommerce-services' ) &&
-			activePlugins.includes( 'jetpack' ) &&
-			profileItems.plugins !== 'already-installed'
-		) {
-			recordEvent(
-				'wcadmin_storeprofiler_already_installed_plugins',
-				{}
-			);
-
-			updateProfileItems( { plugins: 'already-installed' } );
```

Comment:
This code was never happening due to profileItems not being loaded. I've confirmed also that we no longer care about `wcadmin_storeprofiler_already_installed_plugins`